allenact/main.py

```diff
@@ -254,6 +254,15 @@ def get_argument_parser():
         " tutorial https://allenact.org/tutorials/distributed-objectnav-tutorial/",
     )
 
+    parser.add_argument(
```

Comment:
For all changes here, let's discuss if we don't prefer to have it as part of `ExperimentConfig`. Other than that, LGTM.

Reply:
Both work well, I think the question is how modular we want this to be. Do we envision people mixing/matching callbacks across different experiments/runs or are the set of callbacks for a particular experiment config essentially fixed? If the latter then we should just define the set of callbacks in the `ExperimentConfig`, otherwise we will probably want to have this command line option.

One thing that's perhaps missing in this command line version is the ability to set a default set of callbacks for a particular experiment config (i.e. maybe someone wants to always use wandb and doesn't want to have to specify `--callback wandb_callback.py` every time they run an experiment.

Reply:
> One thing that's perhaps missing in this command line version is the ability to set a default set of callbacks for a particular experiment config

If we do this, then aren't we just hard coding the callbacks in the experiment config, and then it might crash for somebody who is running it without permission?